www/widgets/DemoExampleMaker.js

```diff
@@ -134,6 +147,21 @@ class DemoExampleMaker extends Widget {
         <textarea name="dem-url"></textarea>
       </div>
     `
+
+    this._text = new Netitor ({
+      ele: 'div[name="dem-s-text"]',
+      code: 'in this step...',
+      wrap: true,
+      hint: false,
+      lint: false,
+      theme: NNE.theme,
+      language: 'html'
+    })
+    
```

Comment:
there are some "space" characters here on line 160 (let's delete them). u might not notice that b/c VSCode hides space characters by default (these are considered "invisible" characters) but there's a way to visualize them in most editors [including VSCode](https://stackoverflow.com/questions/30140595/show-whitespace-characters-in-visual-studio-code) to avoid adding accidental spaces